c_binding/libsg.c

```diff
@@ -170,12 +218,52 @@ struct _sg_t10_sense_dp {
     /* We don't care the rest of data */
 };
 
+struct _sg_t10_ata_status_return_dp_hdr {
+    uint8_t descriptor_code;
+    uint8_t additional_desc_len;
+};
+
+struct _sg_t10_ata_status_return_dp {
+    uint8_t descriptor_code;
+    uint8_t additional_desc_len;
+    uint8_t rsvd : 7;
+    uint8_t extend : 1;
+    uint8_t error;
+    uint8_t reserved_count;
+    uint8_t count;
+    uint8_t reserved_lba_low;
+    uint8_t lba_low;
+    uint8_t reserved_lba_mid;
+    uint8_t lba_mid;
+    uint8_t reserved_lba_high;
+    uint8_t lba_high;
+    uint8_t device;
+    uint8_t status;
+};
+
 struct _sg_t10_mode_para_hdr {
     uint16_t mode_data_len_be;
     uint8_t we_dont_care_0[4];
     uint16_t block_dp_header_len_be;
 };
 
+struct _sg_t10_log_para_hdr {
+    uint8_t we_dont_care_0[2];
+    uint16_t log_data_len_be;
+};
+
+struct _sg_t10_informational_exceptions_mode_page_0_hdr {
```

Comment:
If possible, please name this struct to `struct _sg_t10_info_excep_mode_page_0_hdr`.